bindings/rust/integration/Cargo.toml

```diff
@@ -6,8 +6,8 @@ edition = "2018"
 publish = false
 
 [dependencies]
-s2n-tls = { version = "0.0.2", path = "../s2n-tls", features = ["testing"] }
-s2n-tls-sys = { version = "0.0.2", path = "../s2n-tls-sys" }
+s2n-tls = { version = "=0.0.4", path = "../s2n-tls", features = ["testing"] }
+s2n-tls-sys = { version = "=0.0.4", path = "../s2n-tls-sys" }
```

Comment:
since this crate isn't published, can you just remove the version from these dependencies?

Reply:
yea, didnt realize we could just do that. :+1: